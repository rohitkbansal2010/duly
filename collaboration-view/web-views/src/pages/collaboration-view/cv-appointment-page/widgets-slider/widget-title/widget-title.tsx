import { ReactNode } from 'react';

import styles from './widgets-title.scss';

type WidgetTitleProps = {
	children?: ReactNode;
};

export const WidgetTitle = ({ children }: WidgetTitleProps): JSX.Element =>
  (
    <h2 className={styles.cvWidgetTitle}>{children}</h2>
  );
