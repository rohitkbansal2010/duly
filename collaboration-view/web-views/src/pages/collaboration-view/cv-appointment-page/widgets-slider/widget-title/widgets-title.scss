@import "~@styles/typography";
@import "~@styles/colors";

.cvWidgetTitle {
  @include nexa-h2-bold;
  @include nexa-extra-bold-highlight;
  color: $dupage-dark-blue;

  font-size: var(--font-size-62px-4px);
  line-height: 6rem;
}
