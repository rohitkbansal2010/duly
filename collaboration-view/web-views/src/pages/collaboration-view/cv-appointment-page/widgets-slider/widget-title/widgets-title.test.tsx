import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { WidgetTitle } from './widget-title';

describe('WidgetTitle', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<WidgetTitle>Mock Title</WidgetTitle>);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
