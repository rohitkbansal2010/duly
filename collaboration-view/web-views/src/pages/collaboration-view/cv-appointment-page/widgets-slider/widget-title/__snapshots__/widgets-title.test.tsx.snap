// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WidgetTitle should match snapshot 1`] = `
<h2
  className="cvWidgetTitle"
>
  Mock Title
</h2>
`;
